src/Cartography/Cartography.DynamicMap/Platforms/Android/Clustering/ClusterRenderer.cs

```diff
@@ -0,0 +1,56 @@
+﻿#if __ANDROID__
+using Android.Content;
+using Android.Gms.Maps;
+using Android.Gms.Maps.Model;
+using Android.Gms.Maps.Utils.Clustering;
+using Android.Gms.Maps.Utils.Clustering.View;
+using Android.Gms.Maps.Utils.UI;
+using Android.Graphics.Drawables;
+using Android.Widget;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Runtime.Remoting.Contexts;
+using System.Text;
+using Windows.UI.Xaml;
+using Context = Android.Content.Context;
+
+namespace Cartography.DynamicMap.Platforms.Android.Clustering
```

Comment:
name space should be Cartography.DynamicMap and put the namespace in a single line with ;